core/src/libraries/Volatility.sol

```diff
@@ -167,9 +167,9 @@ library Volatility {
         assert(sqrtRatioAX96 <= sqrtRatioX96 && sqrtRatioX96 <= sqrtRatioBX96);
 
         unchecked {
-            uint256 numerator = mulDiv96(sqrtRatioX96, sqrtRatioBX96 - sqrtRatioX96);
+            uint256 numerator = Math.mulDiv(uint256(liquidity) << 128, sqrtRatioX96, sqrtRatioBX96);
 
-            value0 = Math.mulDiv(liquidity, numerator, sqrtRatioBX96);
+            value0 = mulDiv224(numerator, sqrtRatioBX96 - sqrtRatioX96);
             value1 = mulDiv96(liquidity, sqrtRatioX96 - sqrtRatioAX96);
```

Comment:
Should match the 2nd branch of the version of this function that's in `LiquidityAmounts`. Copied here because in this case, we don't care about the other branches.